sonar-slang-plugin/src/test/java/com/sonarsource/slang/externalreport/ExternalReportTestUtils.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * SonarSource SLang
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonarsource.slang.externalreport;
+
+import com.sonarsource.slang.kotlin.SlangPlugin;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.sonar.api.SonarQubeSide;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.batch.fs.internal.TestInputFileBuilder;
+import org.sonar.api.batch.sensor.Sensor;
+import org.sonar.api.batch.sensor.internal.SensorContextTester;
+import org.sonar.api.batch.sensor.issue.ExternalIssue;
+import org.sonar.api.internal.SonarRuntimeImpl;
+import org.sonar.api.utils.Version;
+import org.sonar.api.utils.log.LogTester;
+import org.sonar.api.utils.log.LoggerLevel;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public final class ExternalReportTestUtils {
+
+  private ExternalReportTestUtils() {
+    // utility class
+  }
+
+  public static List<ExternalIssue> executeSensor(Sensor sensor, SensorContextTester context) {
+    sensor.execute(context);
+    return new ArrayList<>(context.allExternalIssues());
+  }
+
+  public static SensorContextTester createContext(Path projectDir, int majorVersion, int minorVersion) throws IOException {
+    SensorContextTester context = SensorContextTester.create(projectDir);
+    Files.list(projectDir)
+      .forEach(file -> addFileToContext(context, projectDir, file));
+    context.setRuntime(SonarRuntimeImpl.forSonarQube(Version.create(majorVersion, minorVersion), SonarQubeSide.SERVER));
+    return context;
+  }
+
+  public static void assertLogsContainOnlyInfo(LogTester logTester) {
```

Comment:
name doesn't correspond to content of method

Reply:
renamed to assertNoErrorWarnDebugLogs